frontend/src/containers/Homepage/Homepage.css

```diff
@@ -16,3 +16,10 @@
   margin-top: 20px;
   text-align: center;
 }
+
+h2 {
+	
+	line-height: 1rem;
```

Comment:
Since the project is using "px" as units, I would change the rem units. Circleci is showing it as a Continuous Integration error.